mesa/Person.py

```diff
@@ -266,8 +267,43 @@ def find_job(self):
             self.my_job = the_job
             the_job.my_worker = self
 
+    def factors_c(self):
+        """
+        This procedure modifies the attribute self.criminal_tendency in-place, based on the characteristics of the agent.
+        [employment, education, propensity, crim-hist, crim-fam, crim-neigh, oc-member]
+        :return: None
+        """
+        # employment
+        self.criminal_tendency *= 1.30 if self.job_level == 1 else 1.0
+        # education
+        self.criminal_tendency *= 0.94 if self.education_level >= 2 else 1.0
+        # propensity
+        self.criminal_tendency *= 1.97 if self.propensity > (np.exp(
+            self.m.nat_propensity_m - self.m.nat_propensity_sigma ** 2 / 2) + self.m.nat_propensity_threshold * np.sqrt(
+            np.exp(self.m.nat_propensity_sigma) ** 2 - 1) * np.exp(
+            self.m.nat_propensity_m + self.m.nat_propensity_sigma ** 2 / 2)) else 1.0
+        # crim-hist
+        self.criminal_tendency *= 1.62 if self.num_crimes_committed >= 0 else 1.0
+        # crim-fam
+        self.criminal_tendency *= 1.45 if self.family_link_neighbors() and (
+                len([agent for agent in self.family_link_neighbors() if agent.num_crimes_committed > 0]) / len(
+            self.family_link_neighbors())) > 0.5 else 1.0
+        # crim-neigh
+        self.criminal_tendency *= 1.81 if self.get_neighbor_list("friendship") or self.get_neighbor_list("professional") and (
+                len([agent for agent in self.get_neighbor_list("friendship") if agent.num_crimes_committed > 0]
+                    + [agent for agent in self.get_neighbor_list("professional") if agent.num_crimes_committed > 0]) / len(
+            [agent for agent in self.get_neighbor_list("friendship")] + [agent for agent in self.get_neighbor_list(
+                "professional")])) > 0.5 else 1.0
+        # oc-member
+        self.criminal_tendency *= 4.50 if self.oc_member and not (
+                    self.m.intervention_is_on() and self.m.oc_members_scrutinize) else 1.0
```

Comment:
let's get rid of `and not (self.m.intervention_is_on() and self.m.oc_members_scrutinize)`. It is useless.